supabase/pending/1_single_ops_collection.sql

```diff
@@ -0,0 +1,47 @@
+begin;
+
+-- Rename the existing ops tenant to the new specific ops tenant.
+update tenants set tenant = 'ops.us-central1.v1/' where tenant = 'ops/';
+
+-- Rename the current catalog stats table to preserve its historical stats data.
+alter table catalog_stats owner to postgres;
```

Comment:
I think the table rename, as well as dropping flow_checkpoints & flow_materializations ought to wait until 2_create_stats_table.sql

I'm assuming you've run this migration locally, I'm a bit surprised that deleting `flow_materializstions_v2` doesn't cause a publication error when deleting the older `ops/tenant/catalog-stats-view` materializations, but :shrug:

Reply:
The materializations will still delete if the metadata tables aren't there (or their spec isn't in the metadata tables), but if they can't find the metadata, they won't do anything else to the endpoint: https://github.com/estuary/connectors/blob/03951232c66e203fec123c3dd1bcb6661bf9ecd8/materialize-sql/driver.go#L235-L239

I've just realized that this means they won't actually try to delete their target table if the metadata tables are already deleted, so we can indeed re-create the `catalog_stats` table in this same transaction without the deletion publications bothering it.